ur_controllers/src/force_torque_sensor_controller.cpp

```diff
@@ -0,0 +1,180 @@
+//
+// Created by livanov93 on 12/12/20.
+//
+
+#include "ur_controllers/force_torque_sensor_controller.h"
```

Comment:
Did this went through `ament_cpplit` without any problems

it seems to me that at least the following is missing:
```suggestion
#include "ur_controllers/force_torque_sensor_controller.h"

#include <string>
#include <vector>
#include <unordered_map>

#include "geometry_msgs/msg/vector3.hpp"
#include "hardware_interface/loaned_command_interface.hpp"
```